### DISCUSSION TITLE
How to run a script on deno deploy with --unstable and --no-check flags?

### DISCUSSION THREAD

**Discussion by l3onh4rd** [denoland/deno#13607] [Q&A] [answered]
Hi Deno team,

the title sums it up. I would like to run a script on deno deploy but the build/deployments fails. Locally it only runs with --unstable and --no-check flags. Is there a possibility to run script on deno deploy with these flags.

Kind regards,

Leo

**Comment by kitsonk** [comment, marked as answer]
You should ask this at: https://github.com/denoland/deploy_feedback/issues

But there is no type checking that occurs on Deploy so `--no-check` is meaningless.  If you are trying to access unstable APIs without feature detecting them on Deploy, the code is simply unsupported.  There is no concept of an `--unstable` flag.

**Reply by l3onh4rd** [reply]
Thank you for your quick answer.
